### ISSUE TITLE
removing hunspell-en breaks dependency 'hunspell-en' required by antergos-common-meta

### ISSUE THREAD

**Issue by MaxXor** [Antergos/antergos-packages#372]
```
:: Replace hunspell-en with extra/hunspell-en_AU? [Y/n] Y
:: Replace hunspell-en with extra/hunspell-en_CA? [Y/n] Y
:: Replace hunspell-en with extra/hunspell-en_GB? [Y/n] Y
:: Replace hunspell-en with extra/hunspell-en_US? [Y/n] Y
error: failed to prepare transaction (could not satisfy dependencies)
:: removing hunspell-en breaks dependency 'hunspell-en' required by antergos-common-meta
Error installing repo packages
```

**Comment by nonumeros** [comment]
You don't have a choice. One recommendation would be to **IgnorePkg** during updates, then go ahead with the updates. Then remove **hunspell-en** which will remove **antergos-common-meta** in turn. 

Once done with the above go ahead with the updates again. This was tested to work.

**Comment by MaxXor** [comment]
This should be fixed in `antergos-common-meta` package.

**Comment by nonumeros** [comment]
>This should be fixed in antergos-common-meta package.

@MaxXor  I can't agree more with you. But as I'm sure you know, this issue happened a few weeks ago. 
Back then I was thinking on the same lines. But I thought what the heck. By the time a new release comes up all of this will be removed. Back to normal. But I was wrong. It was never removed. Or hunspell-en at the very least.

Pacman should be less ambiguous too

**Comment by nonumeros** [comment]
>sudo pacman -R antergos-common-meta

@karasu I don't think that's possible until after you resolve the conflict first. On the other hand what you just suggested works as long as you go back a couple of weeks back or whenever it happened the first time around. 

With a new installation of say KDE this is not possible unless you go around it. Don't tell me it's the case because yes, I tried it, to no avail. I had to resolve the conflict first before continuing. Of course, everyone knows by now that the conflict is hunspell-en with the rest . As a matter of fact, the rest here means **hunspell-en_GB**, **hunspell-en_US**, and so forth..

IN other words, it may be as simple to remove `antergos-common-meta` had it been a few weeks ago, but not any longer.

Also, I was taken back after the fact that during installation, all of a sudden pacman wanted to include all the hunspell-en_* packages rather than only one or two.
